migrations/20231114175920_new-payment-methods.sql

```diff
@@ -0,0 +1,28 @@
+ALTER TABLE users
+    ADD COLUMN paypal_country text NULL,
+    ADD COLUMN paypal_email text NULL,
+    ADD COLUMN paypal_id text NULL,
+    ADD COLUMN venmo_handle text NULL,
+
+    DROP COLUMN midas_expires,
+    DROP COLUMN is_overdue,
+    DROP COLUMN stripe_customer_id,
+    DROP COLUMN payout_wallet,
+    DROP COLUMN payout_wallet_type,
+    DROP COLUMN payout_address;
+
+ALTER TABLE historical_payouts
+    RENAME TO payouts;
+
+ALTER TABLE payouts
+    ADD COLUMN method text NULL,
+    ADD COLUMN method_address text NULL,
+    ADD COLUMN platform_id text NULL,
+    ADD COLUMN fee numeric(40, 20) NULL,
+    ALTER COLUMN id TYPE bigint,
+    ALTER COLUMN id DROP DEFAULT;
+
+UPDATE payouts
+SET status = 'success';
```

Comment:
is this intended to hit every past payout?

Reply:
Yes by default the backend assumes them as successes